sentry-backtrace/src/integration.rs

```diff
@@ -18,7 +18,7 @@ pub struct ProcessStacktraceIntegration;
 impl ProcessStacktraceIntegration {
     /// Creates a new Integration to process stacktraces.
     pub fn new() -> Self {
-        Self::default()
+        Self {}
```

Comment:
actually, unit structs don’t need braces, I’m surprised noone complains about that :-D